vllm/worker/model_runner.py

```diff
@@ -147,18 +148,21 @@ def _prepare_prompt(
         input_tokens = _make_tensor_with_pad(input_tokens,
                                              max_prompt_len,
                                              pad=0,
-                                             dtype=torch.long)
+                                             dtype=torch.long,
+                                             device=self.device)
```

Comment:
Can we use the `_set_default_torch_device` context manager here to not repeat `device=self.device`?

Reply:
After second thoughts, I found that explicitly specifying the devices would be better as we might mix CPU and accelerator in some cases.